proto/metadata.proto

```diff
@@ -395,6 +395,22 @@ message Metadata {
     // Device doesn't support router advertisement enable and mode config
     // Juniper: b/316173974 
     bool ipv6_router_advertisement_config_unsupported = 138;
+    // Skip gNMI container OP tc.
+    bool skip_container_op = 139;
```

Comment:
Can we create a new ticket in [https://partnerissuetracker.corp.google.com](https://partnerissuetracker.corp.google.com) to track all these deviations? and add that ticket id here as a comment. 

The comment would be of the format `// Cisco: https://partnerissuetracker.corp.google.com/issues/<issue id>`

Reply:
Cisco: https://partnerissuetracker.corp.google.com/issues/322291556